google-cloud-clients/google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -1476,6 +1476,35 @@ public void testCopyJob() throws InterruptedException, TimeoutException {
     assertTrue(remoteTable.delete());
   }
 
+  @Test
+  public void testCopyJobWithLabels() throws InterruptedException {
```

Comment:
I think this test can be simplified to be focused on labels. I mean you can remove unaffected things from this test case so that it becomes easy to follow.

Reply:
Done.